src/utils/__tests__/fixtures/incidents.json

```diff
@@ -135,5 +135,102 @@
     },
     "incident_date_start": "2018-11-29T23:03:19+01:00",
     "text_extra": ""
+  },
+  {
+    "_links": {
+      "self": {
+        "href": "https://acc.api.data.amsterdam.nl/signals/v1/private/signals/6343"
+      }
+    },
+    "_display": "6343 - m - None - 2020-10-20T13:47:34.399504+02:00",
+    "id": 6343,
+    "signal_id": "28f79e0a-fab7-4c48-a1b6-8681de13d138",
+    "source": "online",
+    "text": "Wow, er vaart iemand te hard onder de Berlagebrug door, die BTW stuk is en er ligt ook nog eens een grote plas olie op het waterrr.",
+    "text_extra": "",
+    "status": {
+      "text": null,
+      "user": null,
+      "state": "m",
+      "state_display": "Gemeld",
+      "target_api": null,
+      "extra_properties": null,
+      "send_email": false,
+      "created_at": "2020-10-20T13:47:34.419776+02:00"
+    },
+    "location": {
+      "id": 7109,
+      "stadsdeel": "M",
+      "buurt_code": null,
+      "area_type_code": null,
+      "area_code": null,
+      "address": {
+        "postcode": "1097DS",
+        "huisletter": "",
+        "huisnummer": 159,
+        "woonplaats": "Amsterdam",
+        "openbare_ruimte": "Weesperzijde",
+        "huisnummer_toevoeging": "1"
+      },
+      "address_text": "Weesperzijde 159-1 1097DS Amsterdam",
+      "geometrie": {
+        "type": "Point",
+        "coordinates": [
+          4.91404056,
+          52.34718534
+        ]
+      },
+      "extra_properties": {
+        "original_address": {
+          "postcode": "1097DS",
+          "huisnummer": "159-1",
+          "woonplaats": "Amsterdam",
+          "openbare_ruimte": "Weesperzijde"
+        }
+      },
+      "created_by": null,
+      "bag_validated": true
+    },
+    "category": {
+      "sub": "Overige boten",
+      "sub_slug": "overig-boten",
+      "main": "Overlast op het water",
+      "main_slug": "overlast-op-het-water",
+      "category_url": "https://acc.api.data.amsterdam.nl/signals/v1/public/terms/categories/overlast-op-het-water/sub_categories/overig-boten",
+      "departments": "ASC, WAT",
+      "created_by": null,
+      "text": null
+    },
+    "reporter": {
+      "email": "",
+      "phone": "",
+      "sharing_allowed": false
+    },
+    "priority": {
+      "priority": "normal",
+      "created_by": null
+    },
+    "type": {
+      "code": "REQ",
+      "created_at": "2020-10-20T16:19:48.860161+02:00",
+      "created_by": "a.tudorache@amsterdam.nl"
+    },
+    "created_at": "2020-10-20T13:47:34.399504+02:00",
+    "updated_at": "2020-10-20T16:19:43.364251+02:00",
+    "incident_date_start": "2020-10-20T13:47:34+02:00",
+    "incident_date_end": null,
+    "operational_date": null,
+    "has_attachments": true,
+    "extra_properties": null,
+    "notes": [],
+    "directing_departments": [
+      {
+        "id": 6,
+        "code": "ASC",
+        "name": "Actie Service Centrum",
+        "is_intern": true
+      }
+    ],
+    "has_children": true
```

Comment:
The `has_children` is the only addition, right? I'd just add that property to some of the already existing incident fixtuers.